fastlane/Fastfile

```diff
@@ -59,6 +59,95 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases"
+lane :update_native_dependencies_to_latest do |options|
+  latest_ios_release = get_latest_github_release_name(repo_name: "RevenueCat/purchases-ios")
+  latest_android_release = get_latest_github_release_name(repo_name: "RevenueCat/purchases-android")
+
+  UI.message("ℹ️  Latest RevenueCat iOS version: #{latest_ios_release}")
+  UI.message("ℹ️  Latest RevenueCat Android version: #{latest_android_release}")
+
+  update_ios_native_version(version: latest_ios_release)
+  update_android_native_version(version: latest_android_release)
+
+  run_pod_install
+end
+
+desc "Update purchases-ios dependency version"
+lane :update_ios_native_version do |options|
+  dry_run = options[:dry_run]
+  if options[:version]
+    new_version_number = options[:version]
+  else
+    UI.user_error!("Missing `version` argument")
+  end
+
+  previous_version_number = parse_previous_ios_native_version
+
+  if previous_version_number == new_version_number
+    UI.message("ℹ️  RevenueCat iOS is already in its latest version: #{previous_version_number}")
+    next
```

Comment:
I would use `return` here and the android version

Reply:
I don't think it's possible to call return like that in Ruby, it gives a `LocalJumpError`. This next will skip the rest of the execution and go back to the line that executed. According to their docs https://docs.fastlane.tools/advanced/lanes/


The next keyword can be used to stop executing a lane before it reaches the end.

lane :build do |options|
  if cached_build_available?
    UI.important 'Skipping build because a cached build is available!'
    next # skip doing the rest of this lane
  end
  match
  gym
end

private_lane :cached_build_available? do |options|
  # ...
  true
end
When next is used during a lane switch, control returns to the previous lane that was executing.

lane :first_lane do |options|
  puts "If you run: `fastlane first_lane`"
  puts "You'll see this!"
  second_lane
  puts "As well as this!"
end

private_lane :second_lane do |options|
  next
  puts "This won't be shown"
end
When you stop executing a lane early with next, any after_each and after_all blocks you have will still trigger as usual :+1:

Reply:
Ahh I didn't know that about fastlane. Makes sense!